lib/field_mapper/standard/field.rb

```diff
@@ -157,7 +157,7 @@ def boolean(value)
       def time(value)
         return value.utc if value.is_a?(Time)
         return value.to_time.utc if value.is_a?(Date)
-        return value.to_time(:utc) if value.is_a?(String)
+        return value.to_time(:utc) rescue nil if value.is_a?(String)
```

Comment:
Crazy you're with PF now. It sure is a small world. Lets add the same rescue to line 159 & another test for that as well. Then I'll get this merged. Thanks @mogman1

Reply:
Is there a way for a Date object to fail to convert to a Time object, and/or a Time object to fail to convert to UTC?

Reply:
Oh doh... you're right. I didn't look closely enough. Thanks.